tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot

```diff
@@ -82,6 +82,28 @@ Docker volume create with specific capacity
     Should Be Equal As Integers  ${rc}  0
     Should Be Equal As Strings  ${disk-size}  96.0G
 
+Docker volume create large volume specifying units
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=unitVol1 --opt Capacity=10G
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal As Strings  ${output}  unitVol1
+    Set Suite Variable  ${ContainerName}  unitContainer
+    Run  docker ${params} run --name ${ContainerName} -d -v ${output}:/mydata busybox /bin/df -Ph
+    ${ContainerRC}  ${output}=  Run And Return Rc And Output  docker ${params} wait ${ContainerName}
+    Should Be Equal As Integers  ${ContainerRC}  0
+    Should Not Contain  ${output}  Error response from daemon
+    ${disk-size}=  Run  docker ${params} logs ${ContainerName} | grep by-label | awk '{print $2}'
+    Should Be Equal As Strings  ${disk-size}  9.5G
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=unitVol2 --opt Capacity=10000
```

Comment:
Minor comment - an empty line in-between test "units" would improve readability.